client/go/internal/vespa/target.go

```diff
@@ -14,6 +14,7 @@ import (
 	"strings"
 	"time"
 
+	"github.com/fatih/color"
```

Comment:
Needs to weigh adding a dependency vs. defining a constant for the ANSI color here. 🤔

Reply:
`github.com/fatih/color` is already a dependency!